react/components/QuoteDetails/QuoteDetails.tsx

```diff
@@ -196,6 +216,20 @@ const QuoteDetails: FunctionComponent = () => {
     })
       .then((result: any) => {
         if (result.data.createQuote) {
+          const metricsParam = {
+            sessionProfile: sessionResponse?.namespaces
+              ?.profile as SessionProfile,
+            accountName: sessionResponse?.namespaces?.account?.accountName
+              ?.value as string,
+            userData: userData?.getUserByEmail?.[0],
+            costCenterName: costCenterData?.getCostCenterById?.name,
+            buyOrgName: organizationData?.getOrganizationById?.name,
```

Comment:
I'm not 100% sure,  but it's possible that all this data is already present in the session or segment cookie, so it would be good to double check before making the changes I suggested to the data fetching.